sdk/storage/azure-storage-queue/azure/storage/queue/_shared/request_handlers.py

```diff
@@ -71,7 +71,11 @@ def get_length(data):
             pass
         else:
             try:
-                return fstat(fileno).st_size
+                mode = fstat(fileno).st_mode
```

Comment:
This was a bugfix for blob (upload_blob): https://github.com/Azure/azure-sdk-for-python/pull/19725 should be fine to propogate elsewhere?

Reply:
Yep, should be fine. Looks like a customer made this change and only added to blob.